tests/system/python/packages/test_multiple_assets.py

```diff
@@ -203,32 +203,38 @@ def test_multiple_assets_with_restart(self, remove_and_add_pkgs, reset_fledge, s
                 on endpoint GET /fledge/asset"""
 
         total_benchmark_services = 6
-        total_assets = PER_BENCHMARK_ASSET_COUNT * total_benchmark_services
+        num_assets_per_service = (num_assets//total_benchmark_services)
+        total_assets = num_assets_per_service * total_benchmark_services
 
         for count in range(total_benchmark_services):
             service_name = BENCHMARK_SOUTH_SVC_NAME + "{}".format(count + 1)
-            add_benchmark(fledge_url, service_name, count + 1)
+            add_benchmark(fledge_url, service_name, count + 1, num_assets_per_service)
             verify_service_added(fledge_url, service_name)
-
+        
         # Wait until total_assets are created
-        time.sleep(PER_BENCHMARK_ASSET_COUNT + 2 * wait_time)
+        time.sleep(num_assets_per_service + 2 * wait_time)
```

Comment:
More deterministic wait can be added by checking the number of assets created using while loops.